src/traces/scattergeo/attributes.js

```diff
@@ -114,6 +114,17 @@ module.exports = overrideAll({
     marker: extendFlat({
         symbol: scatterMarkerAttrs.symbol,
         opacity: scatterMarkerAttrs.opacity,
+        angle: scatterMarkerAttrs.angle,
+        angleref: extendFlat({}, scatterMarkerAttrs.angleref, {
+            values: ['previous', 'up', 'north'],
+            description: [
+                'Sets the reference for marker angle.',
+                'With *previous* the angles are based on the previous position of points on a line.',
+                'With *up* the angles are based on the current positions of points towards up.',
+                'With *north* the angles are based on the current positions of points towards north.',
```

Comment:
```suggestion
                'With *previous*, angle 0 points along the line from the previous point to this one.',
                'With *up*, angle 0 points toward the top of the screen.',
                'With *north*, angle 0 points north based on the current map projection.',
```
(same for `scatter`, I just commented here because it has the most options)